src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -396,19 +474,30 @@ class WP_HTML_Tag_Processor {
 	/**
 	 * Specifies mode of operation of the parser at any given time.
 	 *
-	 * | State         | Meaning                                                              |
-	 * | --------------|----------------------------------------------------------------------|
-	 * | *Ready*       | The parser is ready to run.                                          |
-	 * | *Complete*    | There is nothing left to parse.                                      |
-	 * | *Incomplete*  | The HTML ended in the middle of a token; nothing more can be parsed. |
-	 * | *Matched tag* | Found an HTML tag; it's possible to modify its attributes.           |
+	 * | State           | Meaning                                                              |
+	 * | ----------------|----------------------------------------------------------------------|
+	 * | *Ready*         | The parser is ready to run.                                          |
+	 * | *Complete*      | There is nothing left to parse.                                      |
+	 * | *Incomplete*    | The HTML ended in the middle of a token; nothing more can be parsed. |
+	 * | *Matched tag*   | Found an HTML tag; it's possible to modify its attributes.           |
+	 * | *Text node*     | Found a #text node; this is plaintext and modifiable.                |
+	 * | *CDATA node*    | Found a CDATA section; this is modifiable.                           |
+	 * | *Comment*       | Found a comment or bogus comment; this is modifiable.                |
```

Comment:
Similar to above, I'd be consistent here:

```suggestion
	 * | *Text node*     | Found a TEXT_NODE; this is plaintext and modifiable.                |
	 * | *CDATA node*    | Found a CDATA_SECTION_NODE ; this is modifiable.                           |
	 * | *Comment node*  | Found a COMMENT_NODE or bogus comment; this is modifiable.                |
```

These could also become `#text`, `#cdata-section`, `#comment`, but I prefer the former because it also lets us say `DOCUMENT_TYPE_NODE` and `DOCUMENT_NODE` (whose `nodeName` seems to be `html` and `HTML` respectively and unhelpfully).

Sorry for mangling the nice table alignment 🙂